pkg/rpcserver/rpcserver.go

```diff
@@ -275,6 +292,8 @@ func (serv *Server) handshake(conn *flatrpc.Conn) (string, []byte, *cover.Canoni
 			infoReq.Error = err.Error()
 		}
 	}
+	kaslrOffset := backend.GetKaslrOffset(modules, serv.PCBase)
```

Comment:
Let's combine these 2 functions into 1:
```
modules = backend.FixModules(serv.LocalModules, modules, serv.PCBase)
```
It looks like simpler and more stable interface for users of the backend package.